### PR TITLE
Update info for slow commands

### PR DESCRIPTION
**Pull request by andy9775**
Per help from the slack channel, exporting the BUFFALO_PLUGIN_PATH env variable fixes the issue of slow command line performance.

### CODE REVIEW

File: templates/docs/troubleshooting.html [hunk 1]
```diff
@@ -7,6 +7,10 @@
 After a recent change in the [github.com/gorilla/sessions](http://www.gorillatoolkit.org/pkg/sessions) Buffalo applications will fail to start with the error `securecookie: hash key is not set`. To fix this you must set an environment variable named `SESSION_SECRET`.
 
 For information see [https://github.com/gobuffalo/buffalo/issues/1067](https://github.com/gobuffalo/buffalo/issues/1067)
+    <% } %>
+      
+     <%= faq("Command line is slow", {name: "slow-commands"}) { %>
+If executing `buffalo --help` or any other command from the terminal takes longer than expected set `export BUFFALO_PLUGIN_PATH=$GOPATH/bin` in your shell config (e.g. .bash_profile).
```
Comment (robbyoconnor) [inline templates/docs/troubleshooting.html]: The idea is to narrow the search path -- my `$PATH` is pretty insane...

**Comment by stanislas-m** [comment]
@andy9775 What version of Buffalo do you use? Since v0.11.0 (https://github.com/gobuffalo/buffalo/commit/d024ca0b1ebcc6750aa93d6759f342db86b82e51), if this var is not set, it defaults to the `$GOPATH/bin`, and should be faster.

**Comment by andy9775** [comment]
@stanislas-m I'm on 0.11.1. 

Setting `BUFFALO_PLUGIN_PATH` did make it noticeably faster though.

**Comment by stanislas-m** [comment]
@andy9775 Sorry for the (long) delay, and thanks!

**Pull request merged**